app/livechat/client/views/app/tabbar/customerChatHistory.html

```diff
@@ -0,0 +1,40 @@
+<template name='customerChatHistory'>
+	<div class="content">
+		<div class="list-view">
+			<div>
+				<div class="active-pink-4 mb-4 inner-addon right-addon">
+					<i class="glyphicon glyphicon-search icon-search"></i>
+					<input type="text" id="searchInput"  class="" placeholder="Search" aria-label="Search">
+				</div>
+			</div>
+			{{#if hasChatHistory}}
+				{{#if isAllChat}}
+				<div class="vistior-history-block">
+					<ul id='allist'>
+					{{#if isSearching}}
+						{{#if isfound }}
+							{{#each chat in searchResults}} 
+								{{> chatRoomSearchItem chat}}
+							{{/each}}
+						{{else}}
+							<h1 class='noresult'>No result found</h1> 
```

Comment:
You need to add the translated string value here instead of a hardcoded "No result found".
```suggestion
							<h1 class='noresult'>{{_ "No_results_found"}}</h1> 
```

Reply:
done

Reply:
done